source/patterns/02-organisms/global/site-header~en.yaml

```diff
@@ -0,0 +1,4 @@
+logo:
+  image:
+    src: ../../images/logo.svg
```

Comment:
This was working before https://github.com/libero/pattern-library/pull/83. Not sure why the reference no longer works. A full reference will work but would like to resolve this.

Reply:
Pattern Lab no longer reads the `source` directory directly. I've pushed 1ec3410 to fix.